<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddLogic</class>
 <widget class="QDialog" name="AddLogic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>680</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>125</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Input</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Output</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FuzzyValue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FuzzyAnd</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FuzzyOr</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pid</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Timer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Map</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>381</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>30</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="fguts">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>991</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>{</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>640</y>
     <width>57</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>}</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>641</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="returnval">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>void </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="declaration">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string> noname </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="parameter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>(void)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="Min">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>681</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1022</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="MinLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>671</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Expected Real World Minimum</string>
   </property>
  </widget>
  <widget class="QLabel" name="MaxLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>681</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Expected Real World Maximum</string>
   </property>
  </widget>
  <widget class="QSlider" name="Max">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>681</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1023</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QDial" name="dial">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>50</y>
     <width>61</width>
     <height>64</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1024</number>
   </property>
   <property name="singleStep">
    <number>16</number>
   </property>
   <property name="pageStep">
    <number>64</number>
   </property>
   <property name="value">
    <number>64</number>
   </property>
   <property name="sliderPosition">
    <number>64</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="ScaleLabel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>57</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddLogic</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddLogic</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
